SofaKernel/extlibs/newmat/CMakeLists.txt

```diff
@@ -56,4 +56,4 @@ endif()
 
 
 include(../../SofaFramework/SofaMacros.cmake)
-sofa_create_package(Newmat 0 ${PROJECT_NAME} ${PROJECT_NAME})
+sofa_generate_package(NAME Newmat VERSION 0 TARGETS ${PROJECT_NAME} INCLUDES ${PROJECT_NAME})
```

Comment:
Do you really need to pass the argument parameter names: "NAME, VERSION, ..." ?
In the cmake function I use I directly give the parameter values.

Reply:
This is the whole point of the PR actually :)
Unnamed arguments are ambiguous, as nobody ever really knows what they are for, without taking a look at the macros themselves, which nearly no one does.

Reply:
ok

Reply:
INCLUDES_ROOT_DIR